molecule/command/init/scenario.py

```diff
@@ -182,5 +208,12 @@ def scenario(ctx, dependency_name, driver_name, lint_name, provisioner_name,
     if verifier_name == 'ansible':
         command_args['verifier_lint_name'] = 'ansible-lint'
 
+    if driver_template is not None:
```

Comment:
?

```python
driver_template = driver_template or os.environ.get('MOLECULE_SCENARIO_DRIVER_TEMPLATE', None)
if driver_template:
    command_args['driver_template'] = driver_template
```

Reply:
This implements the cli argument precedence proposed by @ssbarnea in issue #2117. There is also a test case for this:

```python
def test_command_init_scenario_custom_template_precedence
```

Reply:
I keep the same precedence? Fine though, I just can't resist to suggest removing an `if/else` ;)